apps/ledger-live-mobile/src/screens/PTX/Earn/EarnInfoDrawer.tsx

```diff
@@ -19,31 +24,45 @@ export function EarnInfoDrawer() {
     await dispatch(setEarnInfoModal({}));
     await setModalOpened(false);
   }, [dispatch]);
-  const { message, messageTitle } = useSelector(earnInfoModalSelector);
+  const { message, messageTitle, learnMoreLink } = useSelector(earnInfoModalSelector);
 
   useEffect(() => {
     if (!modalOpened && (message || messageTitle)) {
       openModal();
     }
   }, [openModal, message, messageTitle, modalOpened]);
 
+  const onLearnMorePress = useCallback(() => {
+    if (learnMoreLink) {
+      Linking.openURL(learnMoreLink);
+    }
+  }, [learnMoreLink]);
+
+  const { colors } = useTheme();
+
   return (
     <QueuedDrawer isRequestingToBeOpened={modalOpened} onClose={closeModal}>
-      <Flex rowGap={52}>
+      <Flex rowGap={32}>
         <Track onMount event="Earn Info Modal" />
-        <Flex rowGap={56}>
-          <Flex rowGap={16}>
-            <Text variant="h4" fontFamily="Inter" textAlign="center" fontWeight="bold">
-              {messageTitle}
-            </Text>
-            <Text variant="body" lineHeight="21px" color="neutral.c70" textAlign="center">
-              {message}
-            </Text>
-          </Flex>
+        <Flex rowGap={16} alignItems="center">
+          <Circle size={64} bg={colors.opacityDefault.c05}>
+            <Icons.InformationFill size={"L"} color={colors.opacityDefault.c80} />
```

Comment:
Instead of importing useTheme you can just do color={"opacityDefault.c80"}.

Reply:
I personally prefer the type-safe guarantees of theme over non-literal strings. But I'm happy to make the change if this is the convention.